### ISSUE TITLE
Pain at the anterior aspect of the elbow

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10817]
**Preferred term label:**
Pain at the anterior aspect of the elbow

**Synonyms**
pain on the side of the elbow that bends in

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0030835

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal biceps tendon tear
Biceps tendonitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001142
- label: Pain at the anterior aspect of the elbow
- parents: [Elbow pain (HP:0030835)](https://hpo.jax.org/app/browse/term/HP:0030835)
- pmid: 
- def: An unpleasant sensation characterized by physical discomfort (such as pricking, throbbing, or aching) localized to the front part of the elbow front part of the elbow, that is, on the ventral side of the arm where the arm bends.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10817
- synonyms:
